Scene Management System + User Interface

lim-at-infinity opened the pull request.

lim-at-infinity committed:
Transitioned from SteamVR to Oculus

lim-at-infinity committed:
Saving files before refreshing line endings

lim-at-infinity committed:
Converted to MRTK and made AR scene

lim-at-infinity committed:
Updated .gitignore

lim-at-infinity committed:
Created initial User Interface

- User is presented panels that can change the settings of a GIBS object

lim-at-infinity committed:
Start Screen Complete

- Currently allows main menu to different scenes
- Utilizes mainmenu.cs & returntomainmenu.cs via Unity SceneManager

lim-at-infinity committed:
User Interface Completed + Added Start Menu Scene

jtroberts commented:
There are several user specific references to "C:\Users\Angel\" that need to be removed.

- MSBuildForUnity.Common.props
- MSBuild/
- Assets/Libraries/MRTK/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/DotNetAdapter/.obj 

These are all files or directories that probably should be in gitignore.

lim-at-infinity committed:
Update .gitignore

grabermtw committed:
Added Coordinates.cs

Coordinates.cs contains a static class Coordinates with static methods allowing for conversion between Unity XYZ world coordinates and latitude and longitude and back.

lim-at-infinity committed:
Created airsFile and airsParser files

lim-at-infinity committed:
Merge pull request #3 from grabermtw/master

Added Coordinates.cs

lim-at-infinity committed:
Updated AIRS files

lim-at-infinity committed:
Added CSV parser to read AIRS data

lim-at-infinity committed:
Fixed variable datatype in airFile.cs
